cdap-proto/src/main/java/io/cdap/cdap/proto/sourcecontrol/RepositoryConfig.java

```diff
@@ -31,8 +31,11 @@ public class RepositoryConfig {
   private final String pathPrefix;
   private final AuthConfig auth;
 
-  private RepositoryConfig(Provider provider, String link, String defaultBranch, AuthConfig authConfig,
-                           @Nullable String pathPrefix) {
+  private RepositoryConfig(Provider provider,
+                          String link,
+                          String defaultBranch,
```

Comment:
is this also Nullable?

Reply:
yes it should be. It's @Nullable in develop branch now

Reply:
@arjan-bal There seems to be some conflicts introduced by my last PR merge, pls rebase to `develop` and resolve them, thanks!

Reply:
Default branch is nullable from latest discussion on design. We take the remote HEAD if branch is unspecified. Rebase done.